docs/reference/ImageMath.rst

```diff
@@ -31,20 +31,21 @@ Example: Using the :py:mod:`~PIL.ImageMath` module
               b=im2
             )
 
-.. py:function:: lambda_eval(expression, options)
+.. py:function:: lambda_eval(expression, options, **kw)
 
     Returns the result of an image function.
 
     :param expression: A function that receives a dictionary.
     :param options: Values to add to the function's dictionary, mapping image
-                    names to Image instances. You can use one or more keyword
-                    arguments instead of a dictionary, as shown in the above
-                    example. Note that the names must be valid Python
-                    identifiers.
+                    names to Image instances. Note that the names must be
+                    valid Python identifiers. Deprecated.
+                    You can instead use one or more keyword arguments, as
+                    shown in the above example.
+    :param **kw: Values to add to the function's dictionary.
```

Comment:
It seems reasonable to add long description to actual argument and leave only short version for deprecated one.

Reply:
I've updated the commit, moving some text around.